resources/views/child/show.blade.php

```diff
@@ -34,12 +34,11 @@
                         <div>
                             {{ $content->description }}
                         </div>
-                    </td>
                     <td><img style="max-width:200px;max-height:100px;" src="{{ $content->image }}"></td>
                     <td class="edit_delete">
                         <div>
                             <p>
-                                <a href="{{ url('child/edit/' . $content->id) }}">Edit</a>
+                                <a href="{{ url('child/edit/artwork/anecdote/' . $content->id) }}">Edit</a>
```

Comment:
Vanwaar de aanpassing naar /artwork/anecdote/ ? dit lijkt op een route die niet klopt en voor 2 types geld. Laten we dit ook even bespreken.